.github/workflows/build-manylinux-container-images.yml

```diff
@@ -20,7 +20,7 @@ on:  # yamllint disable-line rule:truthy
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
@Qalthos

This won't even start the jobs anymore because GH has fully sunset 20.04 a few weeks ago.

*(hence https://github.com/ansible/pylibssh/pull/708)*

But I can tell you that when I last updated the PR and was able to see the logs, they didn't look any different and were hitting segfaults in those QEMU jobs (cancelling the rest).

Reply:
I am successfully building the containers (or, most of them at least) locally, and I'm trying to get this in a shape where GH is at least running them so I can see what the differences are

Reply:
Okay, feel free to push to this PR. I'll avoid updating it meanwhile.